index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const Headers = require('./src/containers/Headers')
+const Manifest = require('./src/containers/Manifest')
+const Pager = require('./src/containers/Pager')
+const PageSizer = require('./src/containers/PageSizer')
+const Rows = require('./src/containers/Rows')
+const Status = require('./src/containers/Status')
```

Comment:
We should probably use `import` here to be consistent, we are going to need babel anyway.